app/gui2/src/components/VisualizationContainer.vue

```diff
@@ -49,7 +56,7 @@ function blur(event: Event) {
 const rootNode = ref<HTMLElement>()
 const contentNode = ref<HTMLElement>()
 
-onMounted(() => (config.width = Math.max(config.nodeSize.x, MIN_WIDTH_PX)))
+onMounted(() => (config.width = Math.max(config.width ?? config.nodeSize.x, MIN_WIDTH_PX)))
```

Comment:
This is also a bit of strange mechanic: we update width in config (effectively updating it also in code's metadata if I read code properly) on mounting container, i.e. when showing visualization, possibly with the width the node happen to have ATM. I think it's better when the metadata keeps the width set by user by resizing.

Reply:
The purpose of this code is a bit different:
If we have with from metadata (`config.width`), then
  - Set current width to at least `MIN_WIDTH_PX`
Else
  - Set current width and metadata width to current node size, or MIN_WIDTH_PX, whichever is larger.

Without this change, we won’t *load* the width initially, we would always use `config.nodeSize.x`.